hack/nfpm.yml

```diff
@@ -0,0 +1,30 @@
+name: docker-buildx
```

Comment:
Do we have a convention for package names? I see docker scan is [named `docker-scan-plugin`](https://github.com/docker/docker-ce-packaging/blob/8bbb124d84a94765592fcb4aeef66982daf2d69d/deb/common/control#L105). Maybe we should do the same here with `docker-buildx-plugin`? cc @thaJeztah

Reply:
Based on https://github.com/docker/cli/pull/3314#issuecomment-945667368 we are distancing from "plugin" lingo so would rather avoid it. The bundled tools by Docker org that make up "docker" are better referred as components. Compose is compose, not compose-plugin. Same for buildx.